notebook/Solution/04 - Hybrid-CNN-LSTM Univariate.ipynb

```diff
@@ -0,0 +1,1292 @@
+      "text/plain": [
+       "<Figure size 720x432 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# Create Learining Curve \n",
```

Comment:
1. spelling error
2. The graph looks abit unusual for a loss curve, suggest to start from 1st epoch onwards to illustrate better?